checkstyle-tester/diff.groovy

```diff
@@ -180,14 +180,10 @@ def generateCheckstyleReport(cfg) {
     executeCmd("git checkout $cfg.branch", cfg.localGitRepo)
     executeCmd("git log -1 --pretty=MSG:%s%nSHA-1:%H", cfg.localGitRepo)
 
-    def testerCheckstyleVersion = getCheckstyleVersionFromPomXml('./pom.xml', 'checkstyle.version')
-    def checkstyleVersionInLocalRepo = getCheckstyleVersionFromPomXml("$cfg.localGitRepo/pom.xml", 'version')
-    if (testerCheckstyleVersion != checkstyleVersionInLocalRepo) {
-        throw new GroovyRuntimeException("Error: config version mis-match!\nCheckstyle version in tester's pom.xml is $testerCheckstyleVersion\nCheckstyle version in local repo is $checkstyleVersionInLocalRepo")
-    }
+    def checkstyleVersion = getCheckstyleVersionFromPomXml("$cfg.localGitRepo/pom.xml", 'version')
 
     executeCmd("mvn -Pno-validations clean install", cfg.localGitRepo)
-    executeCmd("groovy launch.groovy --listOfProjects $cfg.listOfProjects --config $cfg.checkstyleCfg --ignoreExceptions --ignoreExcludes")
+    executeCmd("groovy launch.groovy --listOfProjects $cfg.listOfProjects --config $cfg.checkstyleCfg --ignoreExceptions --ignoreExcludes --checkstyleVersion" + checkstyleVersion)
```

Comment:
Aren't you missing a space after `checkstyleVersion`?

Reply:
yes, thanks